README.md

```diff
@@ -41,6 +50,11 @@ Logged in.
 
 At this point, you are able to issue queries.
 
+If you need to log in from a CI context, it is possible to set tjhe
+environment variables `MORGUE_USERNAME` and `MORGUE_PASSWORD`. If these
```

Comment:
do you think b64 encoded secrets (like k8s?) is a common enough use case to add support for it preemptively?

Reply:
I'm not aware of a CI platform which will take an environment variable secret and give it back to you in anything but a plaintext form, and I don't see people trying to run morgue in an automated fashion otuside that kind of environment, so I vote for leaving this until someone asks for it.